== TEST (2019-05-14 16:37:15) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:37:17) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma,10))
-- EXECUTION TIME:
0.127997
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:37:18) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma,10))
-- EXECUTION TIME:
0.130909
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:37:18) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma,10))
-- EXECUTION TIME:
0.161694
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:37:18) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma,10))
-- EXECUTION TIME:
0.180171
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:37:18) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma,10))
-- EXECUTION TIME:
0.297061
-- ERROR:
sh: 1: cannot open /home/codebench/codes/1716/2505/cbtemp.txt: No such file

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:37:19) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:37:19) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:38:05) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resiltado,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:07) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resiltado,10))
-- EXECUTION TIME:
0.23593
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:07) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resiltado,10))
-- EXECUTION TIME:
0.145323
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:24) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.201067
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:25) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.208965
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:25) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.296673
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:25) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.181962
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:25) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.315791
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:38:25) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:26) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.136225
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:38:26) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:27) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.132221
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:49) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))

contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.14953
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:38:50) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))

contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:38:51) 
-- CODE:
x = eval(input("valor de x"))
k = int(input(""))

contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.120218
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:39:01) 
-- CODE:
x = eval(input("valor de x"))
k = int(input("n de k"))

contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.156335
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:39:02) 
-- CODE:
x = eval(input("valor de x"))
k = int(input("n de k"))

contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:39:03) 
-- CODE:
x = eval(input("valor de x"))
k = int(input("n de k"))

contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.228486
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:43:47) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.20435
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:43:52) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.140631
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:43:53) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:43:53) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.213774
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:43:54) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(soma, 10))
-- EXECUTION TIME:
0.131487
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:16) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado, 10))
-- EXECUTION TIME:
0.154551
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:17) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado, 10))
-- EXECUTION TIME:
0.215926
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:44:18) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:18) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado, 10))
-- EXECUTION TIME:
0.138117
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:18) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado, 10))
-- EXECUTION TIME:
0.166063
-- ERROR:
sh: 1: cannot open /home/codebench/codes/1716/2505/cbtemp.txt: No such file

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:44:18) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:44:18) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:19) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado, 10))
-- EXECUTION TIME:
0.175702
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:36) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado,10))
-- EXECUTION TIME:
0.206215
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:48) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	print(round(resultado,10))
-- EXECUTION TIME:
0.121686
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:56) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.163352
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:57) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while (contador < k):
	resultado = resultado + (sinal(x**expoente/factorial(expoente)))
	sinal = sinal*(-1)
	expoente = expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.154586
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado = resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:45:21) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.157606
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado= resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:45:23) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.13338
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado= resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:45:24) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.135816
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado= resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:45:24) 
-- CODE:

from math import*

x = eval(input("valor de x"))
k = int(input("n de k"))
contador = 1
sinal = -1
resultado = x
expoente = 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.232175
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado= resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:45:47) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.12816
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado= resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:45:48) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 12, in <module>
    resultado= resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:45:48) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.151438
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado= resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:45:48) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.213659
-- ERROR:
sh: 1: cannot open /home/codebench/codes/1716/2505/cbtemp.txt: No such file

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:45:49) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.209193
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 12, in <module>
    resultado= resultado + (sinal(x**expoente/factorial(expoente)))
TypeError: 'int' object is not callable

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:03) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.226341
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:04) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.175558
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:04) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.162589
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:05) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.09883
-- ERROR:
sh: 1: cannot open /home/codebench/codes/1716/2505/cbtemp.txt: No such file

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:05) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.128286
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:06) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.138975
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:06) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.188288
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.8558007816
0.8662952838
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:

-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:06) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.276298
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.8558007816
0.8662952838
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:

-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:22) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.134118
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:46:23) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- OUTPUT:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:24) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.243681
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:24) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.415027
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:24) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.574735
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:24) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.649246
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
0.8558007816
0.8662952838
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:

-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:24) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado,10))
-- EXECUTION TIME:
0.462109
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
0.8558007816
0.8662952838
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:

-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:31) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado, 10))
-- EXECUTION TIME:
0.107095
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:32) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado, 10))
-- EXECUTION TIME:
0.207287
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:57) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado, 10))
-- EXECUTION TIME:
0.143625
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:46:58) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado, 10))
-- EXECUTION TIME:
0.132315
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9248322293
-1.0045248555
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8558007816
0.8662952838
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:47:46) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
	
	print(round(resultado, 10))
-- OUTPUT:
0.9248322293
1.0045248555
0.9998431014
1.0000035426
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:48:52) 
-- CODE:

from math import*

x= eval(input("valor de x"))
k= int(input("n de k"))
contador= 1
sinal= -1
resultado= x
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2
	contador +=1
print(round(resultado, 10))
-- EXECUTION TIME:
0.131429
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8662952838
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
